lib/assessment.rb

```diff
@@ -34,7 +46,9 @@ def every_even(numbers)
   # Write ruby code that returns an array with all of the
   # entries from the words enumerable capitalized.
   def transform(words)
-    # Your code goes here
+    words = words.split(" ").each {|word| word.capitalize!}
```

Comment:
`words` isn't a string, there's no reason to `split(" ")` or `join(" ")`. You're right to use `capitalize`. What operation would we use to transform each element in words, using `capitalize` as our rule?